x-pack/plugins/uptime/server/kibana.index.ts

```diff
@@ -66,11 +62,14 @@ export const initServerWithKibana = (server: UptimeCoreSetup, plugins: UptimeCor
       read: {
         app: ['uptime', 'kibana'],
         catalogue: ['uptime'],
-        api: ['uptime-read', 'actions-read', 'actions-all', 'alerting-read', 'alerting-all'],
+        api: ['uptime-read', 'actions-read', 'actions-all'],
         savedObject: {
           all: ['alert', 'action', 'action_task_params'],
           read: [umDynamicSettings.name],
         },
+        alerting: {
+          all: ['xpack.uptime.alerts.tls', 'xpack.uptime.alerts.monitorStatus'],
+        },
```

Comment:
A note for Uptime - As you granted `all` privileges before I kept it as `all` here, but if you'd like, you can change this to `read` once this is merged, so  users with `read` to Uptime will also have `read` to alerting under Uptime.

Reply:
This would be my preference before feature freeze. I wouldn't expect read access to uptime to allow full CRUD to uptime alerts

Reply:
That's fair, though - it's true at the moment as well, so nothing would change.

Reply:
> This would be my preference before feature freeze. I wouldn't expect read access to uptime to allow full CRUD to uptime alerts

++ I think it's rather awkward that read-only access to Uptime, Metrics and Logs allow the user to CRUD the associated alerts.